src/test/java/com/amazon/ion/impl/EncodingDirectiveCompilationTest.java

```diff
@@ -30,6 +30,8 @@
 
 /**
  * Tests that Ion 1.1 encoding directives are correctly compiled from streams of Ion data.
+ *
+ * Don't attempt fix this without ensuring that MacroCompilerTest and IonManagesWriter_1_1_Test are working first.
```

Comment:
What does this mean?

Reply:
Oh, that was a note to myself that I should resolve failing tests in the other test classes before dealing with the tests in this class. I'll remove it.